src/shared/components/community/community.tsx

```diff
@@ -22,7 +22,7 @@ import {
 } from "@utils/helpers";
 import type { QueryParams } from "@utils/types";
 import { RouteDataResponse } from "@utils/types";
-import { Component, linkEvent } from "inferno";
+import { Component, RefObject, createRef, linkEvent } from "inferno";
```

Comment:
I'm not sure why this additional `ref` stuff is necessary?

Reply:
This was set in the app component for this page, so had to move it in this component to be able to link it in the "main" element that was moved from app to community.
This can only be set on main elements for some reason, or it will be considered an error.

This is used for the "skip to main content" feature.